product-and-services/index.md

```diff
@@ -0,0 +1,13 @@
+# Product and Services
+
+Product and Services at 2i2c is responsible for guiding the evolution of our products and services, and for representing the needs and potential impact of the communities we serve.
+The engineering sub-team works as part of the broader Product and Services team to define ways in which we aim to improve our services, prioritize deliverables according to their impact and executing on delivery and implementation.
```

Comment:
```suggestion
The engineering sub-team works as part of the broader Product and Services team to define ways in which we aim to improve our services, prioritize deliverables according to their impact, and execute on delivery and implementation.
```

Reply:
Oops! Looks like I didn't commit the suggestion before merging. Suggestion added in https://github.com/2i2c-org/team-compass/commit/cb2f8fa4a20c298111ec8c78534da1fe9e33e611 instead.